client/profile-wizard/steps/business-details.js

```diff
@@ -377,7 +379,28 @@ class BusinessDetails extends Component {
 						accountRequiredText
 					) }
 				</p>
-			</Fragment>
+				{ this.bundleInstall && (
+					<p>
+						{ interpolateComponents( {
+							mixedString: __(
+								'By installing Jetpack and WooCommerce Services plugins for free you agree to our {{link}}Terms of Service{{/link}}.',
+								'woocommerce-admin'
+							),
+							components: {
+								link: (
+									// eslint-disable-next-line jsx-a11y/anchor-has-content
+									<a
```

Comment:
Why does this need to be an anchor instead of a `<Link/>` component like in https://github.com/woocommerce/woocommerce-admin/pull/5096?

Reply:
So I was getting this with `Link`:

```
Uncaught Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined
```

Turns out I was trying to use `Link` from `@wordpress/components` whereas I should have been using the one from `@woocommerce/components`. This is fixed now, thanks for the catch.